auxx/src/Command/Tx.hs

```diff
@@ -12,41 +12,66 @@ module Command.Tx
 
 import           Universum
 
-import           Control.Concurrent.STM.TQueue (newTQueue, tryReadTQueue, writeTQueue)
-import           Control.Exception.Safe (Exception (..), try)
-import           Control.Monad.Except (runExceptT)
-import           Data.Aeson (eitherDecodeStrict)
+import           Control.Concurrent.STM.TQueue
+    (newTQueue, tryReadTQueue, writeTQueue)
+import           Control.Exception.Safe
+    (Exception (..), try)
+import           Control.Monad.Except
+    (runExceptT)
+import           Data.Aeson
+    (eitherDecodeStrict)
 import qualified Data.ByteString as BS
-import           Data.Default (def)
+import           Data.Default
+    (def)
 import qualified Data.HashMap.Strict as HM
-import           Data.List ((!!))
+import           Data.List
+    ((!!))
```

Comment:
That is a tool applying overly simple and in my opinion poorly thought out rules.

Reply:
Yeah, I'd agree that the exact logic is ... bad here, though I like the "concept" of it.